oak_runtime/src/node/http/mod.rs

```diff
@@ -49,13 +50,14 @@ use futures_util::{
     future::TryFutureExt,
     stream::{StreamExt, TryStreamExt},
 };
+use maplit::hashmap;
 use prost::Message;
 use tokio_rustls::TlsAcceptor;
 
 #[cfg(test)]
 pub mod tests;
 
-/// Checks that prot is not reserved (i.e., is greater than 1023).
+// Checks that prot is not reserved (i.e., is greater than 1023).
```

Comment:
nit (and precedes this PR) but
```suggestion
// Checks that port is not reserved (i.e., is greater than 1023).
```